core/src/Microsoft.Mcp.Core/Areas/Server/Commands/ServiceStartCommand.cs

```diff
@@ -123,7 +123,6 @@ private IHost CreateHost(ServiceStartOptions serverOptions)
             .ConfigureLogging(logging =>
             {
                 logging.ClearProviders();
-                logging.ConfigureOpenTelemetryLogger();
```

Comment:
Open telemetry logging is server-generic.  Servers that provide non-azure-centric tools would still want to log telemetry.  Servers developed by Microsoft would want to send that telemetry to Azure App Insights.   While this isn't technically "Azure agnostic" it should still be considered Microsoft.Mcp.Core functionality.

Each mcp server we create based on Microsoft.Mcp.Core may need to point to a different app insights instance, so the open telemetry confuguration should come from the server rather than some static defaults in a Microsoft.Mcp.Core config class.